packages/ui/components/Footer.vue

```diff
@@ -1,23 +1,106 @@
 <template>
-  <div class="footer">
-    <WitnetLogo class="witnet-logo white" />
-    <ul class="links-container">
-      <a class="link" :href="documentationUrl" target="_blank">
-        {{ $t('footer.documentation') }}
-      </a>
-      <a class="link" :href="requestDataFeedUrl" target="_blank">
-        {{ $t('footer.request_data_feed') }}
-      </a>
-      <a class="link" :href="witnetUrl" target="_blank">
-        {{ $t('footer.about_witnet') }}
-      </a>
-    </ul>
-    <p class="copyright">{{ $t('footer.copyright') }}</p>
+  <div class="footer-background">
```

Comment:
We can derive all this links from a map. I made a [commit](https://github.com/Tommytrg/data-feed-app/blob/footer/packages/ui/components/footer/Footer.vue) to do it